<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WH</class>
 <widget class="QDialog" name="WH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>工时量统计助手</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,2,0,1">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
         </property>
         <property name="text">
          <string>选择数据目录</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_browser">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Courier New&quot;;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_brsw">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
         </property>
         <property name="text">
          <string>浏览</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,1">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
         </property>
         <property name="text">
          <string>选择统计月份</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbx_months">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Courier New&quot;;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="btn_run">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
         </property>
         <property name="text">
          <string>统计</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,8">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
         </property>
         <property name="text">
          <string>分组:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_group">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;新宋体&quot;;
color: rgb(24, 74, 255);</string>
         </property>
         <property name="text">
          <string>..</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
         </property>
         <property name="text">
          <string>数据路径:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_savedata">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;新宋体&quot;;
color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string>....</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
